big_tests/tests/login_SUITE.erl

```diff
@@ -39,20 +39,25 @@ all() ->
     [
      {group, login},
      {group, login_scram},
+     {group, configure_sha256},
      {group, login_scram_store_plain},
      {group, messages}
     ].
 
 groups() ->
     G = [{login, [parallel], all_tests()},
          {login_scram, [parallel], scram_tests()},
+         {configure_sha256, [pallarel], configure_sha256_test()},
          {login_scram_store_plain, [parallel], scram_tests()},
          {messages, [sequence], [messages_story, message_zlib_limit]}],
     ct_helper:repeat_all_until_all_ok(G).
 
 scram_tests() ->
     [log_one, log_one_scram, log_one_scram_256].
 
+configure_sha256_test() ->
+    [configure_sha256_log_with_sha256].
```

Comment:
I think it'd be beneficial to have a test case `configure_sha256_log_with_sha1_fails`, what do you think? This could be added in a separate PR, though.